src/lightning_fabric/plugins/precision/precision.py

```diff
@@ -30,7 +30,7 @@ class Precision:
     The class attribute precision must be overwritten in child classes. The default value reflects fp32 training.
     """
 
-    precision: _PRECISION_INPUT = "32"
+    precision: "lf.connector._PRECISION_INPUT" = "32"
```

Comment:
@carmocca Perhaps it should be the other way around. Definition here, and imported in connector

Reply:
Sounds good to me